its/ruling/src/test/java/org/sonar/python/it/PythonExtendedRulingTest.java

```diff
@@ -72,7 +72,8 @@ void test_black() throws IOException {
     SonarScanner build = buildWithCommonProperties("black");
     build.setProperty("sonar.sources", "src");
     build.setProperty("sonar.tests", "tests");
-    build.setProperty("sonar.test.exclusions", "tests/data/async_as_identifier.py");
+    build.setProperty("sonar.test.exclusions", "tests/data/async_as_identifier.py, tests/data/tricky_unicode_symbols.py," +
+      "tests/data/pattern_matching_extras.py,tests/data/pep_572_py39.py,tests/data/pattern_matching_invalid.py");
```

Comment:
At least some of these seem to indicate valid syntax that we fail to parse.
I'm currently investigating them and will create relevant tickets for the cases that make sense.

Reply:
SONARPY-1572 and SONARPY-1573 have been created to handle some unicode characters we fail to parse, as well as to cover unparenthesized walrus in set literals/comprehensions.

I didn't manage to create a reproducer for the `pattern_matching_extras` case that is both valid Python and that we fail to parse (using the offending snippet verbatim raises an invalid syntax error in Python 3.11), so I didn't create a ticket for it.